.air.toml

```diff
@@ -5,9 +5,9 @@ tmp_dir = ".air"
 cmd = "make --no-print-directory backend"
 bin = "gitea"
 delay = 1000
-include_ext = ["go", "tmpl"]
+include_ext = ["go", "tmpl", "css", "js"]
 include_file = ["main.go"]
-include_dir = ["cmd", "models", "modules", "options", "routers", "services"]
+include_dir = ["cmd", "models", "modules", "options", "public", "routers", "services", "templates"]
```

Comment:
We use `air` for watching backend files only, frontend files are watched by webpack, so I don't think it should watch `public`. The `templates` have their own reload mechanism in the backend as well.

Reply:
committed that by accident